app/components/global/CookieNotice/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Box } from '@rebass/grid'
+import { Button } from '@pubsweet/ui'
+import FooterMessage from 'ui/atoms/FooterMessage'
+import NativeLink from 'ui/atoms/NativeLink'
+import Helpers from './helpers'
+
```

Comment:
Just a suggestion ... do we want to reference that this cookie is the one from Journal? @Cooryd

Reply:
I'm not sure I can use the same cookie across both sites as they're on different domains. This one is specific to reviewer.elifesciences.org

Reply:
Ah what I meant was via some sort of comment in way of explanation to a dev.

Reply:
I'll leave a comment that the design and wording are copied from journal